charts/newrelic-infra-operator/README.md

```diff
@@ -7,51 +7,52 @@ This chart will deploy the [New Relic Infrastructure Operator][1], which injects
+| `config.infraAgentInjection.customAttributes[].defaultValue`                    | default value for custom attribute to include.                                                                                                                                                                                                                                                                                               |                                                                                                      |
+| `config.infraAgentInjection.customAttributes[].fromValue`                       | label from which take the value of the custom attribute.                                                                                                                                                                                                                                                                                     |                                                                                                      |
+| `config.infraAgentInjection.agentConfig.image.repository`                       | The infrastructure agent repository for the sidecar container.                                                                                                                                                                                                                                                                              | `newrelic/infrastructure-k8s`                                                                        |
+| `config.infraAgentInjection.agentConfig.image.pullPolicy`                       | The sidecar image pull policy.                                                                                                                                                                                                                                                                                                              | `IfNotPresent`                                                                                       |
+| `config.infraAgentInjection.agentConfig.image.tag`                              | The infrastructure agent repository for the sidecar container.                                                                                                                                                                                                                                                                              | `2.4.0-unprivileged`                                                                                 |
```

Comment:
Perhaps tag should be documented after repository? It makes sense from both logical point of view and from alphabetical order.
```suggestion
| `config.infraAgentInjection.agentConfig.image.tag`                              | The infrastructure agent image tag for the sidecar container.                                                                                                                                                                                                                                                                              | `2.4.0-unprivileged`                                                                                 |
```

Reply:
👍